napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -154,6 +158,10 @@ def __init__(self, viewer):
             2,
             3,
         )
+        self.ndisplayButton.setContextMenuPolicy(Qt.CustomContextMenu)
+        self.ndisplayButton.customContextMenuRequested.connect(
+            partial(open_perspective_popup, self)
+        )
         self.ndisplayButton.setToolTip(
             trans._(
                 "Toggle number of displayed dimensions ({shortcut})",
```

Comment:
@tlambert03 could we perhaps add this info to the tooltip?

>Right click to change the field of view of the camera when in 3D mode

Or similar

That would make it a little bit more discoverable?

Reply:
added